src/interpreter/interpreter.ts

```diff
@@ -116,46 +118,18 @@ export class Interpreter extends AbstractInterpreter {
 
 }
 
+const addDefinitionToREPL = (newDefinition: Class | Singleton | Mixin, interpreter: Interpreter) => {
+  const environment = interpreter.evaluation.environment
+  environment.scope.register([REPL, newDefinition])
+  if (newDefinition.is(Singleton)) {
+    interpreter.evaluation.rootFrame.set(newDefinition.fullyQualifiedName, interpreter.evaluation.instantiate(newDefinition))
+  }
```

Comment:
Con o sin llaves? 😶‍🌫️ 
```suggestion
  if (newDefinition.is(Singleton))
    interpreter.evaluation.rootFrame.set(newDefinition.fullyQualifiedName, interpreter.evaluation.instantiate(newDefinition))
```

Reply:
si me olvido las llaves cómo entro a casa? 😄

Reply:
ahí lo subo sin las llaves